packages/sogrim-app/src/stores/DataStore.ts

```diff
@@ -283,4 +284,14 @@ export class DataStore {
     this.userDetails.modified = true;
     return this.userSettings;
   };
+
+  courseTypeIsAll = (type: string | undefined): boolean => {
```

Comment:
1. For boolean functions the naming convention is `isSomethingTrueOrFalse` rather than `somethingIsTrueOrFalse`
2. The term "course type" is somewhat confusing since a course's type is different than a bank's type, here's a list of terms which I find more understandable for this context: 
`courseBankRule | courseBankType | bankTypeOfCourse | bankType | bankRule `

Reply:
Changed to "IsBankTypeOfCourseAll"